src/libstd/env.rs

```diff
@@ -210,6 +216,12 @@ fn _var(key: &OsStr) -> Result<String, VarError> {
 ///
 /// [`None`]: ../option/enum.Option.html#variant.None
 ///
+/// # Panics
+///
+/// This function may panic if `key` is empty, contains an ASCII equals sign
+/// `'='` or the NUL character `'\0'`, or when the value contains the NUL
+/// character.
```

Comment:
NUL character is repeated twice in both of these?

Reply:
This is copied verbatim from the underlying functions called, which state they may crash when either the key or the value contain a null byte.

Reply:
Oh, I missed the key/value distinction.